src/Traits/FixturesTrait.php

```diff
@@ -156,16 +156,24 @@ public function getClearMySQLDatabaseQuery(array $tables, array $except = ['migr
         return  "{$query} SET FOREIGN_KEY_CHECKS = 1;\n";
     }
 
-    public function prepareSequences(array $tables, array $except = []): void
+    public function prepareSequences(array $except = []): void
     {
         $except = array_merge($this->postgisTables, $this->prepareSequencesExceptTables, $except);
 
-        $query = array_concat($tables, function ($table) use ($except) {
-            if (in_array($table, $except)) {
-                return '';
-            } else {
-                return "SELECT setval('{$table}_id_seq', (select coalesce(max(id), 1) from {$table}), (case when (select max(id) from {$table}) is NULL then false else true end));\n";
-            }
+        $data = app('db.connection')
```

Comment:
let's optimize this operation by using some kind of cache or static variables, tables list may be change only once after the migrate command executed

Reply:
Done